vagrant/boxes.d/99-base-dev.yaml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
These are moving out of the example file? If someone copies that example file they will have a clash of named boxes.

Reply:
ah sorry, good catch, that was just something I was trying out, didn't meant to include in PR. I'm